prerelease/README.md

```diff
@@ -0,0 +1,104 @@
+# GitHub Action: Prerelease Terraform Module
+
+<!-- prettier-ignore-start -->
+<!-- action-docs-description -->
+## Description
+
+< GitHub Action to compute a prerelease version based on the latest release version and the number of commits since the latest release. This will also generate a docker tag based on the computed version if the label `prerelease` is specified on the PR.
```

Comment:
```suggestion
GitHub Action to compute a prerelease version based on the latest release version and the number of commits since the latest release. This will also generate a docker tag based on the computed version if the label `prerelease` is specified on the PR.
```

Reply:
I think that this is generated code.

Reply:
I'll look and see

Reply:
It's really weird that it's there.  I'll remove it and see if it readds it.

Reply:
Oh - it's some failure on the part of parsing the yaml to mkdown. 
```
name: GitHub Action Release and Publish Prerelease Module
description: <
  GitHub Action to compute a prerelease version based on the latest release version and the number of commits since the
  latest release.
```